### PR TITLE
Implement create user command

### PR DESCRIPTION
**Pull request by igoihman**

### CODE REVIEW

File: cmd/ocm/create/user/cmd.go [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+	group      string
+}
+
+var Cmd = &cobra.Command{
+	Use:     "user",
+	Aliases: []string{"users"},
+	Short:   "Configure user access for cluster",
+	Long:    "Configure user access for cluster",
+	Example: `# Add a user to the dedicated-admins group
+  ocm create user myuser --cluster=mycluster --group=dedicated-admins`,
```
Comment (zgalor) [inline cmd/ocm/create/user/cmd.go:38]: maybe add an example of how to create multiple users

File: cmd/ocm/create/user/cmd.go [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+		Get().
+		Send()
+	if err != nil {
+		return fmt.Errorf("Group '%s' in cluster '%s' doesn't exist", args.group, clusterKey)
+	}
+
+	for _, username := range strings.Split(users, ",") {
+		user, err := cmv1.NewUser().ID(username).Build()
+		if err != nil {
+			return fmt.Errorf("Failed to create '%s' user '%s' for cluster '%s'", args.group, username, clusterKey)
```
Comment (zgalor) [inline cmd/ocm/create/user/cmd.go:129]: Does this work `Best Effort`?
We create users until we reach a user that we failed.
On first failure, we stop, and all the users that we succeeded are created.

say I had a list of 3 users and the second one already exists.
So the client will see:
```
> ocm create user u1,u2,u3 --cluster=mycluster --group=dedicated-admins`
{
  "kind": "User",
  "href": "/api/clusters_mgmt/v1/clusters/11111/groups/dedicated-admins/users/u1",
  "id": "u1"
}

Failed to add 'dedicated-admins' user 'u2' to cluster 'mycluster': user 'u2' already exists
```

I wonder if continuing would be better user experience. i.e:
```
> ocm create user u1,u2,u3 --cluster=mycluster --group=dedicated-admins`
{
  "kind": "User",
  "href": "/api/clusters_mgmt/v1/clusters/11111/groups/dedicated-admins/users/u1",
  "id": "u1"
}

Failed to add 'dedicated-admins' user 'u2' to cluster 'mycluster': user 'u2' already exists

{
  "kind": "User",
  "href": "/api/clusters_mgmt/v1/clusters/11111/groups/dedicated-admins/users/u3",
  "id": "u3"
}

```

WDYT?

Comment (igoihman) [reply cmd/ocm/create/user/cmd.go:129]: makes sense, I will change it

File: cmd/ocm/create/user/cmd.go (hunk 1 above)
Comment (vkareh) [reply cmd/ocm/create/user/cmd.go:38]: This is marked as resolved, but I don't see the example

File: cmd/ocm/create/user/cmd.go (hunk 2 above)
Comment (vkareh) [reply cmd/ocm/create/user/cmd.go:129]: This is marked as resolved but I don't see any change (or any commits after this message)

File: cmd/ocm/create/user/cmd.go [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+			Group(args.group).
+			Users().
+			Add().
+			Body(user).
+			Send()
+		if err != nil {
+			fmt.Printf("Failed to add '%s' user '%s' to cluster '%s': %v\n", args.group, username, clusterKey, err)
+			continue
+		}
+		fmt.Printf("Failed to add '%s' user '%s' to cluster '%s': %v\n", args.group, username, clusterKey, err)
```
Comment (vkareh) [inline cmd/ocm/create/user/cmd.go:144]: This message seems wrong

File: cmd/ocm/create/user/cmd.go [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+		}
+		_, err = clusterCollection.Cluster(cluster.ID()).
+			Groups().
+			Group(args.group).
+			Users().
+			Add().
+			Body(user).
+			Send()
+		if err != nil {
+			fmt.Printf("Failed to add '%s' user '%s' to cluster '%s': %v\n", args.group, username, clusterKey, err)
```
Comment (vkareh) [inline cmd/ocm/create/user/cmd.go:141]: This should be `fmt.Errorf`

File: cmd/ocm/create/user/cmd.go (hunk 3 above)
Comment (igoihman) [reply cmd/ocm/create/user/cmd.go]: I forgot to edit the sentence after copy-pasting :) Updated now.

File: cmd/ocm/create/user/cmd.go (hunk 4 above)
Comment (vkareh) [reply cmd/ocm/create/user/cmd.go:141]: Based on offline discussion, let's leave this as `fmt.Printf`

**Pull request merged**